src/pages/Home/Home.jsx

```diff
@@ -4,12 +4,8 @@ import LabelsPopover from '../../components/LabelsPopover';
 import SpendingItem from '../../components/SpendingItem';
 import Button from '../../components/Button';
 import Modal from '../../components/Modal';
-import RadioButtonField from '../../components/RadioButtonField';
-import TextareaField from '../../components/TextareaField';
-import NumberField from '../../components/NumberField';
-import LabelsField from '../../components/LabelsField';
 import PeriodPopover from '../../components/PeriodPopover';
-import DateField from '../../components/DateField';
+import NewSpendingForm from '../../components/NewSpendingForm/NewSpendingForm';
```

Comment:
Again (second or third time), why do you use `/NewSpendingForm/NewSpendingForm`? You have `index.js` file inside `NewSpendingForm` folder. You don't need to specify concrete file.
```js
import What from `../../components/NewSpendingForm`;
```